src/wallet/wallet.cpp

```diff
@@ -4496,12 +4496,16 @@ void CWallet::GetFilteredNotes(
 
         // Filter the transactions before checking for notes
         if (!CheckFinalTx(wtx) ||
-            wtx.GetBlocksToMaturity() > 0 ||
             wtx.GetDepthInMainChain() < minDepth ||
             wtx.GetDepthInMainChain() > maxDepth) {
             continue;
         }
 
+        // Filter coinbase transactions that don't have Sapling outputs
+        if (wtx.IsCoinBase() && wtx.mapSaplingNoteData.empty()) {
```

Comment:
Would this cause transparent notes to be "unfiltered" if they're in the same coinbase tx alongside a Sapling output?

Reply:
I think the answer is: yes, and that's intended.

Reply:
Yep; in this function we only care about Sapling outputs, no matter where they are.

Reply:
Why does the code no longer look at maturity at all? We would previously include coinbase transactions with mature transparent outputs and now we don't, right? (Blocking comment.)

Reply:
Oh, because this function is only for shielded transactions.